docs/zh_cn/common_usage/epoch_to_iter.md

```diff
@@ -0,0 +1,81 @@
+# 从 EpochBased 切换至 IterBased
+
+MMEngine 很多模块默认按照 Epoch 训练模型，例如 ParamScheduler, LoggerHook, CheckPointHook 等，常见的 EpochBasedTraining 配置写法如下：
+
+```python
+param_scheduler = dict(
+    type='MultiStepLR',
+    milestones=[6, 8]
+)
+
+default_hooks = dict(
+    logger=dict(type='LoggerHook'),
+    checkpoint=dict(type='CheckpointHook', interval=2),
+)
+
+train_cfg = dict(
+    by_epoch=True,
+    max_epochs=10,
+    val_interval=2
+)
+
+runner = Runner(
+    model=ResNet18(),
+    work_dir='./work_dir',
+    train_dataloader=train_dataloader_cfg,
```

Comment:
Should `train_dataloader_cfg` also be changed?

Reply:
`InfiniteSampler` will be set automatically in `IterBasedTrainLoop`.